lib/trie/database_test.go

```diff
@@ -387,3 +389,59 @@ func TestTrie_GetFromDB(t *testing.T) {
 		}
 	}
 }
+
+func TestStoreAndLoadWithChildTries(t *testing.T) {
+	testCase := []Test{
```

Comment:
```suggestion
func TestStoreAndLoadWithChildTries(t *testing.T) {
	testCase := []keyValue{
```

also please rename from `testCase` to `keyValues`